### DISCUSSION TITLE
Meeting Agenda 3/5

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#325] [Meeting Agendas/Minutes]
1. Review tasks from previous sprint
2. Sprint Planning
3. Meet with Darian

**Comment by moreno-michael** [comment]
Meeting minutes 3/5

1. Task Review
- Issue #1338 in progress, Issue #1339 in progress
- Issue #1295 in progress
- Issue #1337 in progress, Issue #1409 in progress

2. Sprint Planning
-Issue #1294 assigned to Michael
-Issue #1427 and [Issue #319](https://github.com/dyc3/senior-design/issues/319) assigned to Chris
-Issue #1423 and #1385 assigned to Victor
-Brainstorm solutions or alternatives to Issue #1424, ideally would like to have this done and in production for the Expo

3. Meet with Darian
- "Clean up" report for the final report
- Clarify the project abstract for the expo. Pitch the load balancer as a service with various applications and high scalability
- Consider security and backup service applications
- "Start thinking like a business person"
